docs/adding_packages/sources_and_patches.md

```diff
@@ -61,17 +60,20 @@ more chances of failing because of unexpected errors).
 The Conan Team may ask you to remove more if they are taking a lot of resources. When removing old versions, please follow these considerations:
 
 * keep one version for every major release
-* for the latest major release, at least three versions should be available (latest three minor versions)
+* for the latest major release, at least two versions should be available (latest two minor versions)
 
 Logic associated with removed revisions implies that entries in the `config.yml` and `conandata.yml` files should also be removed. If anyone needs to
 recover them in the future, Git contains the full history and changes can be recovered from it.
 
+Removed versions should not affect other recipes available in the repository. If a recipe depends on a removed version, it should be updated to
+depend on the latest available version.
+
 Please, note that even if those versions are removed from this repository, **the packages will always be accessible in ConanCenter remote**
 associated to the recipe revision used to build them.
 
 ### Adding old versions
 
-We love to hear why in the opening description of the pull requests you need this exact version.
+We would love to hear why in the opening description of the pull requests you need this exact version.
```

Comment:
```suggestion
```

Reply:
`We usually don't add old versions unless there is a specific request for it.`

I would add, a "well motivated request" -

Reply:
Addressed by aefd84be4e951bdbd359f7e3182eee49e674c35c